packages/app/src/app/pages/Pro/pages/WorkspacePlanSelection.tsx

```diff
@@ -148,7 +148,7 @@ export const WorkspacePlanSelection: React.FC<{
 
   React.useEffect(
     function trackingWorkspace() {
-      track('Pro - Change Workspace');
+      track('Pro - Change Team selection');
```

Comment:
@tromika is there any problem we update the names of the events? Can we be prepared for this migration?

Reply:
I discussed this with @smhutch and we can do it for sure. On Amplitude side there are multiple ways to tackle this.